ibis/backends/datafusion/compiler.py

```diff
@@ -0,0 +1,396 @@
+    return df.functions.lpad(arg, length, pad)
+
+
+@translate.register(ops.RPad)
+def rpad(op, expr):
+    arg = translate(op.arg)
+    length = translate(op.length)
+    pad = translate(op.pad)
+    return df.functions.rpad(arg, length, pad)
+
+
+@translate.register(ops.GreaterEqual)
+def ge(op, expr):
+    return translate(op.left) >= translate(op.right)
+
+
+@translate.register(ops.LessEqual)
+def le(op, expr):
+    return translate(op.left) <= translate(op.right)
+
+
+@translate.register(ops.Greater)
+def gt(op, expr):
+    return translate(op.left) > translate(op.right)
+
+
+@translate.register(ops.Less)
+def lt(op, expr):
+    return translate(op.left) < translate(op.right)
+
+
+@translate.register(ops.Equals)
+def eq(op, expr):
+    return translate(op.left) == translate(op.right)
+
+
+@translate.register(ops.NotEquals)
+def ne(op, expr):
+    return translate(op.left) != translate(op.right)
+
+
+@translate.register(ops.Add)
+def add(op, expr):
+    return translate(op.left) + translate(op.right)
+
+
+@translate.register(ops.Subtract)
+def sub(op, expr):
+    return translate(op.left) - translate(op.right)
+
+
+@translate.register(ops.Multiply)
+def mul(op, expr):
+    return translate(op.left) * translate(op.right)
+
+
+@translate.register(ops.Divide)
+def div(op, expr):
+    return translate(op.left) / translate(op.right)
+
+
+@translate.register(ops.FloorDivide)
+def floordiv(op, expr):
+    return df.functions.floor(translate(op.left) / translate(op.right))
+
+
+@translate.register(ops.Modulus)
+def mod(op, expr):
+    return translate(op.left) % translate(op.right)
+
+
+@translate.register(ops.Sum)
+def sum(op, expr):
+    arg = translate(op.arg)
+    return df.functions.sum(arg)
+
+
+@translate.register(ops.Min)
+def min(op, expr):
+    arg = translate(op.arg)
+    return df.functions.min(arg)
+
+
+@translate.register(ops.Max)
+def max(op, expr):
+    arg = translate(op.arg)
+    return df.functions.max(arg)
+
+
+@translate.register(ops.Mean)
+def mean(op, expr):
+    arg = translate(op.arg)
+    return df.functions.avg(arg)
+
+
+def _prepare_contains_options(options):
+    if isinstance(options, ir.AnyScalar):
+        # TODO(kszucs): it would be better if we could pass an arrow
+        # ListScalar to datafusions in_list function
+        return [df.literal(v) for v in options.op().value]
+    else:
+        return translate(options)
+
+
+@translate.register(ops.ValueList)
+def value_list(op, expr):
+    return list(map(translate, op.values))
+
+
+@translate.register(ops.Contains)
+def contains(op, expr):
+    value = translate(op.value)
+    options = _prepare_contains_options(op.options)
+    return df.functions.in_list(value, options, negated=False)
+
+
+@translate.register(ops.NotContains)
+def not_contains(op, expr):
+    value = translate(op.value)
+    options = _prepare_contains_options(op.options)
+    return df.functions.in_list(value, options, negated=True)
+
+
+@translate.register(ops.ElementWiseVectorizedUDF)
```

Comment:
in a future version can you put the ops in the files consistent with other backends e.g. udfs